### PR TITLE
Cron job for nightly runs

### PR DESCRIPTION
**Pull request by sebastianklose**
* Fixes python newbie bug (function overloading)
* Adds cloud build config
* Adds Docker for the nightly cron job, rbac binding and yaml for cron job

### CODE REVIEW

File: evaluation/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Kubernetes based Evaluation Pipeline
+This folder contains files needed to setup a kubernetes based evaluation
+pipeleline on GCP.
+
+## Source tree overview
+
+| File / Folder                    | Description                                                               |
+| :------------------------------- | :------------------------------------------------------------------------ |
+| **`cron`**                       | folder with files for cron job (docker image, k8s configs, ...)           |
```
Comment (pifon2a) [inline evaluation/README.md]: nit: you use periods in most of the lines (description column), but not in all.

File: evaluation/cron/cloud_build.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'eu.gcr.io/robco-klose/eval_nightly', '.']
```
Comment (pifon2a) [inline evaluation/cron/cloud_build.yaml:3]: should your gCloud account be visible here? can we use `robco-cartographer`?

**Review by pifon2a — APPROVED** [review]
just nits

File: evaluation/README.md (hunk 1 above)
Comment (sebastianklose) [reply evaluation/README.md]: fixed ...

File: evaluation/cron/cloud_build.yaml (hunk 2 above)
Comment (sebastianklose) [reply evaluation/cron/cloud_build.yaml:3]: not yet. I'll move everything to the cartographer cloud project soon and will write up a little more in the README.md about the configuration steps required while doing so.

File: evaluation/README.md (hunk 1 above)
Comment (pifon2a) [reply evaluation/README.md]: here as well: `Base docker image for the evaluation (mainly to speed up compilation)   `

File: evaluation/cron/cloud_build.yaml (hunk 2 above)
Comment (pifon2a) [reply evaluation/cron/cloud_build.yaml:3]: ACK

**Pull request merged**